### DISCUSSION TITLE
2023-05-10 - Weekly Hosting Meeting

### DISCUSSION THREAD

**Discussion by javiercasares** [WordPress/hosting-handbook#146] [Weekly Meetings]
# Meeting

/here < meeting type="hosting" > Who’s here to chat about hosting?

👋 Hello! This is the weekly meeting for the WordPress.org Hosting Team, who works with hosts and developers to help improve the WordPress user experience. This is a great time to ask any questions you may have, or visit Make/Hosting for more info. 

🥇 If you participate in the Five for the Future program, drop your WordPress.org username in a thread here 👉

🤗 If you're new here, welcome! You are welcome to introduce yourself if you like in a thread here 👉

-----

## Agenda

📓 ··· Here is the agenda for this meeting ···

**WordPress Community**
- 2021-2022 WordPress Meetup Survey
- 2022 Annual Survey Results & Next Steps
- WordPress 6.2.1 Planning
- Gutenberg 15.7
- Stacks: The first community theme
- 20 Days to Celebrate 20 Years of WordPress, From Blogs to Blocks
- Kim Parsell Memorial Scholarship for WordCamp US 2023

**Hosting Team Time**
- SQLite Database Update
- WP CLI Update
- Contributor Days

**Hosting Team Projects**
- Advanced Administration Handbook
- Host Test Runners

**Open Floor**
- Anything not in the agenda

Would anyone like to add to or change any items on the agenda?

OK, let’s get started with the WordPress Community news.

-----

### :community: ··· 2021-2022 WordPress Meetup Survey ···

[2021-2022 WordPress Meetup Survey: Key Findings](https://make.wordpress.org/community/2023/05/02/2021-2022-wordpress-meetup-survey-key-findings/)

The 2021-2022 WordPress Meetup Survey received 2867 responses, with 94% of respondents likely to recommend WordPress meetups. The top challenges to increasing attendance were scheduling conflicts, lack of interest, and event location, while attendees favored group learning formats, with hybrid online/in-person events of interest.

### :community: ··· 2022 Annual Survey Results & Next Steps ···

[2022 Annual Survey Results & Next Steps](https://wordpress.org/news/2023/05/2022-annual-survey-results-next-steps/)

The 2022 Annual Survey received a comprehensive update. The survey showed increased usage of blocks and the new site editor, while WordPressers choose the platform for being open source, flexible, low risk, cost-effective, and having a positive reputation. WordPress is seen as good or better than other CMS platforms, and its ease of use, flexibility, and plugin options are highly valued.

### :wordpress: ··· WordPress 6.2.1 Planning ···

[WordPress 6.2.1 Planning](https://make.wordpress.org/core/2023/05/03/wordpress-6-2-1-planning/)

[WordPress 6.2.1 RC1 is now available](https://make.wordpress.org/core/2023/05/09/wordpress-6-2-1-rc1-is-now-available/)

WordPress 6.2.1 Release Candidate 1 (RC1) is available for testing! Some ways you can help test this minor release:
- Use the [WordPress Beta Tester](https://wordpress.org/plugins/wordpress-beta-tester/) plugin
- As this is a minor RC release, select the Point Release channel and the Nightlies stream. This is the latest build including the RC and potentially any subsequent commits in [trunk](https://make.wordpress.org/core/handbook/contribute/codebase/).
- Use WP-CLI to test: `wp core update https://wordpress.org/wordpress-6.2.1-RC1.zip`

### :plugins: ··· Gutenberg 15.7 ···

[Gutenberg 15.7](https://make.wordpress.org/core/2023/05/04/whats-new-in-gutenberg-15-7-03-may/)

For the latest release of the Gutenberg plugin, contributors continued improvements to existing UI and UX for content creators, site owners working on their own site, or theme developers creating new themes. Users will find small changes for streamlined workflows, fixed bugs, and refined responsiveness, just to name a few. Highlights: 
- Site Logo replacement via the sidebar
- Duotone filter controls in block sidebar
- Fluid Typography scales giant fonts to small screens

### :themes: ··· Stacks: The first community theme ···

[Stacks: The first community theme](https://make.wordpress.org/themes/2023/05/04/stacks-the-first-community-theme/)

Not a lot to say. The best way is to check the Theme [Stacks](https://wordpress.org/themes/stacks/) and look at the preview… [Stacks preview](https://wp-themes.com/stacks/).

### :community: ··· 20 Days to Celebrate 20 Years of WordPress, From Blogs to Blocks ···

[20 Days to Celebrate 20 Years of WordPress, From Blogs to Blocks](https://make.wordpress.org/marketing/2023/05/02/20-days-to-celebrate-20-years-of-wordpress-from-blogs-to-blocks/)

[Day 1: #WP20 From Blogs to Blocks](https://make.wordpress.org/marketing/2023/05/08/day-1-wp20-from-blogs-to-blocks/)
[Day 2: #WP20 From Blogs to Blocks](https://make.wordpress.org/marketing/2023/05/09/day-2-wp20-from-blogs-to-blocks/)

We are so happy to be celebrating WordPress and the incredible WordPress community as we approach the 20th anniversary of WordPress on May 27, 2023. We hope you enjoy participating in this community game and that it gives you an opportunity to reflect on and share your favorite parts of WordPress. 

### :community: ··· Kim Parsell Memorial Scholarship for WordCamp US 2023 ···

The WordPress Foundation is once again offering the Kim Parsell Memorial Scholarship for travel to [WordCamp US 2023](https://us.wordcamp.org/2023/).

This scholarship is awarded to someone who:
- Identifies as a woman
- Is a WordPress contributor
- Has never attended WordCamp US before
- Requires financial assistance to attend

-----

### :wordpress: ··· SQLite Update ··· 

[[Status update on the SQLite project](https://make.wordpress.org/core/2023/04/19/status-update-on-the-sqlite-project/)]

The initial implementation was included in the [Performance Lab](https://w.org/plugins/performance-lab) plugin and then released as a [stand-alone plugin](https://w.org/plugins/sqlite-database-integration/).

Why should this be in Core and not as a plugin?

In its current form, the SQLite implementation is a plugin. Just like all plugins, it can only be installed on a pre-existing website. As a result, a site can only use an SQLite database if it already has a MySQL database – effectively negating all the benefits that SQLite can bring to WordPress.

Using the featured plugin is a great way to allow users to test the implementation and iron out any issues etc. However, long-term, it doesn’t make sense to use it as a plugin.

The proof-of-concept UI in that implementation checks if the server supports SQLite and MySQL:

- If SQLite is supported but not MySQL, then it defaults to SQLite and no UI option is shown to users. This is for example what happens in the [WordPress Playground](https://playground.wordpress.net/) – it is vanilla WordPress that defaults to SQLite.
- Similarly, if the server supports MySQL and not SQLite, then the installation will use MySQL and no option will be shown to users.
- The UI option will only show if both SQLite and MySQL are supported by the server.

How can this implementation be tested?

The next step would be thoroughly to test the implementation with all the plugins you normally use. You can use the [SQLite Database Integration plugin](https://w.org/plugins/sqlite-database-integration/) to test an SQLite database on your existing website, or better yet – you can test the [pull request in WordPress Core](https://github.com/WordPress/wordpress-develop/pull/3220).

### 🤖 ··· WP-CLI 2.8.0 ···

[WP-CLI v2.8.0 release date](https://make.wordpress.org/cli/2023/04/05/wp-cli-v2-8-0-release-date/)

The v2.8.0 release of WP-CLI is scheduled to be published on 2023-05-03.

Among others a multitude of bug fixes and some new features, we’re also expecting this release to fix all known PHP 8.2 issues within WP-CLI itself.

The release candidate for this release will already be available sometime during the week prior, to allow for early testing. We will announce the availability of that release candidate in the #hosting-community channel to allow hosters to do early smoke testing.

### 🗓️ ···  Hosting Contributor Day ···

Heads up, some events are coming down the pipeline along with some Contributor Days to participate in:

- 2023-06-08, [WordCamp Europe](https://europe.wordcamp.org/2023/), Athens (Greece).
- 2023-08-24, [WordCamp US](https://us.wordcamp.org/2023/), National Harbor, Maryland (USA).

Is anyone [going to a WordCamp](https://central.wordcamp.org/schedule/) and wants to lead a Hosting Table?

-----

Great, so just a look at our projects.

### 📕 ··· Advanced Administration Handbook ···

The Advanced Administration handbook is now public and online.

- [Public Handbook](https://developer.wordpress.org/advanced-administration/)
- [GitHub Handbook](https://github.com/WordPress/Advanced-administration-handbook)

Currently, [everything pending to do is in the Issues part](https://github.com/WordPress/Advanced-administration-handbook/issues). A lot of "documentation + enhancement" tickets, and also some "documentation + good first issue" for Contributor Day.

If anybody needs help or mentoring for this project, @javiercasares is always available.

### :cloud: ··· Host Test Runners ···

You may have heard of [Host Test Results](https://make.wordpress.org/hosting/test-results/), a community-led project that helps WordPress users find the best hosts for their sites. Host Test Results provides a comprehensive list of hosting providers and detailed information about each host's performance and reliability.

We want to make sure that as many hosts as possible are included in the Host Test Results database, so that WordPress users, the #core-test, and the #core team can benefit from this valuable resource. That's why we're asking you to check if your host is already supporting Host Test Results, and to add your host if it still needs to be included.

To check if your hoster is already supporting Host Test Results, visit the website (https://make.wordpress.org/hosting/test-results/) and search for your host's name. If your host is already included, great! You can use the information provided to make sure that you're getting the best possible performance and reliability from your site. 

If your host isn't included, don't worry, you can still help! You can just follow the instructions on the website to add your host to the database. It's quick and easy, and you'll be helping other WordPress users find the best hosts for their sites. However, if your host has not received reports in the last 25 revisions, we encourage you to check back with them.

- [Try the PHPUnit Test Runner](https://make.wordpress.org/hosting/handbook/tests/#try-the-phpunit-test-runner)
- [How to report: Creating your bot for WordPress.org](https://make.wordpress.org/hosting/handbook/tests/#how-to-report-creating-your-bot-for-wordpress-org)

### :cloud: ··· WordPress /hosting page & Project Bedrock ···

[Project Bedrock (aka WordPress /hosting)](https://docs.google.com/document/d/1FhjFjY9I6gn6uqz4oBsHvYzxnrUg0RfAYuRhlLjdWec/)

For a long time, there has been [wordpress.org/hosting](https://wordpress.org/hosting/), which is underused and which is currently just that, a simple page.

For many years, the Make Hosting team has been raising the possibility of creating a directory in which any hosting company that meets a series of predefined requirements, can appear as recommended hosting, or compatible with the WordPress CMS.

This project began as the [“Requirement Matrix” Criteria](https://docs.google.com/document/d/1uvb3LsOyTZ0D2mVw7yokfCxMb0lCkFWdSnXD1b-2HWU/), a list created by several members of the Community in which dozens of factors are listed by which each of these hosting providers can be considered, either by functionalities (not numbers), alignment with the project and WordPress values, the minimum technical requirements and options.

This document will try to solve some doubts regarding what factors should be considered, how to develop the project, what teams should be involved in it, and how to maintain the site.

An important element, which should appear in the list, is respect for the use of the WordPress brand.
This living directory can also help WordPress as a brand to make sure we have a transparent and trusted source of hosting companies. Companies where the onboarding experience is friendly and in a well-maintained shape, to not lose newcomers to other solutions like WIX or Shopify. We, as a community, need to make sure that we don’t lose them as customers and users of our ecosystem only because they couldn’t find a good hosting solution.

-----

## Open Floor

Ok, if nothing else, let’s move on to 🤖 Open Floor. Folks are welcome to bring any topic up for discussion at this time.

-----

## Next meeting

⏰ Here are the [team's upcoming meetings](https://make.wordpress.org/meetings/#hosting):

- 2023-05-10 at 18:00 UTC (in a few hours)
- 2023-05-17 at 09:00 UTC (next week)
- 2023-05-17 at 18:00 UTC

📌 Okay, we're out of time for today, let's close out this meeting.

You are welcome to continue the conversation here in the #hosting-community channel or on Make/Hosting, at any time.

Thank you all for joining and for your contributions to WordPress!
See you at the next Hosting Team weekly meeting! < /meeting >

**Comment by jessfrick** [comment]
The only Five for the Future participant we had in the late meeting was pwtyler. 

I did update the content regarding WP-CLI to reflect the delayed release. I also updated the content for Project Bedrock to include notes from the earlier meeting. 

Thanks for pulling these together @javiercasares !
